### DISCUSSION TITLE
VCOM and LAMBS AI

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1310] [Question and answer]
Hello there, 
I would like to know if it is possible to use VCOM and LAMBS AI with this scenario.

**Comment by Vdauphin** [comment]
Hello,

I don't know, for lamb we talk a little about it here: https://github.com/Vdauphin/HeartsAndMinds/discussions/1077

Cheers

**Reply by Vsauce07** [reply]
Hi, from what I understand it's okay to use LAMBS but no VCOM? I know that VCOM creates waypoints but do they overwrite that of H&M?
